controllers/managedocs_controller.go

```diff
@@ -1048,6 +1059,16 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 			},
 			Key: alertRelabelConfigSecretKey,
 		}
+		// Enabling remote write for only provider clusters
+		if r.DeploymentType == providerDeploymentType {
+			var remoteWriteSpec *promv1.RemoteWriteSpec
+			if remoteWriteSpec, err = r.reconcilePrometheusRemoteWrite(); remoteWriteSpec != nil {
+				r.Log.Info("Prometheus remote write spec fetched , will enable remote write")
+				r.prometheus.Spec.RemoteWrite = append(r.prometheus.Spec.RemoteWrite, *remoteWriteSpec)
+			} else {
+				r.Log.Info("Unable to fetch Prometheus Remote write spec, will disable remote write", "prom-remote-write-spec-error", err)
```

Comment:
- I think you meant
```suggestion
				fmt.Errorf("Unable to fetch Prometheus Remote write spec, will disable remote write", err)
```

Reply:
This is something that would be useful when we later include promtail support to treat error logs by a  metric label.And since the presence of the secret & config map is not mandatory in my IMO an error log message isn't required, just an info that remote write functions are disabled.